src/assignementtwo/controller/SortOptions.java

```diff
@@ -0,0 +1,99 @@
+package assignementtwo.controller;
+
+import assignementtwo.model.Constants;
+import assignementtwo.model.User;
+import assignementtwo.sorting.SortByAddress;
+import assignementtwo.sorting.SortByAge;
+import assignementtwo.sorting.SortByName;
+import assignementtwo.sorting.SortByRollNo;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+import java.util.Set;
+
+/*
+ * This class is use to sort parameter according to choice
+ */
+public class SortOptions {
+
+    /*
+     * the users are stored in form of set for one build
+     */
+    private final Set<User> users;
+
+    /*
+     * the value of choice of item
+     * and the order of choice in ascending or descending
+     */
+    private int choice;
+    private int order;
+
+    /*
+     * parameterized constructor in initialise set
+     * @param us set of user
+     */
+    public SortOptions(final Set<User> users) {
+        this.users = users;
+    }
+
+    /*
+     * Shows the options the user has to enter to do
+     * any particular sorting
+     */
+    private void showdetails() {
+        final Scanner scan = new Scanner(System.in);
```

Comment:
try to use var fearture
```suggestion
        final var scan = new Scanner(System.in);
```

Reply:
Use it other places also

Reply:
done